horreum-backend/src/main/java/io/hyperfoil/tools/horreum/svc/Util.java

```diff
@@ -634,6 +634,9 @@ public static Instant toInstant(Object time) {
          return Instant.ofEpochMilli(((Number) time).longValue());
       } else {
          String str = time.toString().trim();
+         if(str.isBlank()){
+            return null;
+         }
```

Comment:
Do we want to log something here? because that's actually a similar case of `conversion failure`.
Maybe something like `failed to convert null instant to timestamp`, this could be useful foir debugging purposes if not expected.

Reply:
I originally thought any invalid input to the function should be logged (thus the issue being opened) but apparently Horreum has a use case where start and stop are not specified and they default to the currrent system timestamp. In that use case the input to `toInstant` results in a blank (or empty) string string.
To me that means it is an expected input case and should not be logged as a problem. 

If we decide to introduce debug or trace logging for user input then perhaps we could add additional logging then but I don't think we need it for the purpose of properly handling input to `toInstant`.

Reply:
yeah I missed that use case, then I agree with your approach!